### PR TITLE
Creating jumbotron - Selahaddin

### PR DESCRIPTION
**Pull request by justsealan**
## Naming Rules

Name your PR like this: JUMBOTRON-SELAHADDIN

## Description

Create a page header on hero image and sign up form.

## Related to

[Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:
](https://github.com/Haseeb3927/Booster/projects/1#card-79060411)
Fixes #1

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import { Route, Routes } from "react-router-dom";
 
-import About from "./pages/secondPage/About";
-import Home from "./pages/mainPage/components/home/Home.js";
+// import About from "./pages/About";
+import Home from "./pages/Home";
```
Comment (Haseeb3927) [inline client/src/App.js:4]: Please remove this comment

File: client/src/components/Header/Form.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+			margin: theme.spacing(2),
+		},
+	},
+}));
+
+const Form = ({ handleClose }) => {
+	const classes = useStyles();
+	// create state variables for each input
+	// const [Name, setName] = useState("");
+	// const [lastName, setLastName] = useState("");
```
Comment (Haseeb3927) [inline client/src/components/Header/Form.js:28]: Again commented code

File: client/src/components/Header/Form.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+				</Button>
+				<Button type="submit" variant="contained" color="primary">
+					login
+				</Button>
+			</div>
+		</form>
+	);
+};
+
+export default Form;
```
Comment (Haseeb3927) [inline client/src/components/Header/Form.js:66]: Well-intended code 
I'm learning Material Ui as well

File: client/src/components/Jumbotron/HeroLayout.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+					}}
+				/>
+				<Background sx={sxBackground} />
+				{/* <Box
+					component="img"
+					src="/static/themes/onepirate/productHeroArrowDown.png"
+					height="16"
+					width="12"
+					alt="arrow down"
+					sx={{ position: "absolute", bottom: 32 }}
```
Comment (Haseeb3927) [inline client/src/components/Jumbotron/HeroLayout.js:65]: Delete Comment please

File: client/src/components/Jumbotron/Registration.js [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+import SnackbarContent from "@material-ui/core/SnackbarContent";
+import IconButton from "@material-ui/core/IconButton";
+import ErrorIcon from "@material-ui/icons/Error";
+import Typography from "@material-ui/core/Typography";
+import VisibilityTwoToneIcon from "@material-ui/icons/VisibilityTwoTone";
+import VisibilityOffTwoToneIcon from "@material-ui/icons/VisibilityOffTwoTone";
+import CloseIcon from "@material-ui/icons/Close";
+
+
+class Registration extends Component {
```
Comment (Haseeb3927) [inline client/src/components/Jumbotron/Registration.js:19]: Please convert to function component

File: client/src/components/Jumbotron/Registration.js [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+				error: "Passwords don't match",
+			});
+		}
+		const newUserCredentials = {
+			email: this.state.email,
+			password: this.state.password,
+			passwordConfrim: this.state.passwordConfrim,
+		};
+		console.log("this.props.newUserCredentials", newUserCredentials);
+		//dispath to userActions
```
Comment (Haseeb3927) [inline client/src/components/Jumbotron/Registration.js:67]: Delete Comment please

**Review by Haseeb3927 — CHANGES_REQUESTED** [review]
Good indentation also please read the comments left and make changes accordingly

File: client/src/index.css [hunk 7]
```diff
@@ -1,4 +1,8 @@
-
+body {
```
Comment (Haseeb3927) [inline client/src/index.css:1]: I'm not sure if we should keep the CSS I wrote at the start of project ?
Do you need it ?

File: client/src/App.js (hunk 1 above)
Comment (justsealan) [reply client/src/App.js:4]: comments removed

File: client/src/components/Header/Form.js (hunk 2 above)
Comment (justsealan) [reply client/src/components/Header/Form.js:28]: comments removed

File: client/src/App.js [hunk 8]
```diff
@@ -1,12 +1,13 @@
...
 
 const App = () => (
 	<Routes>
+
 		<Route path="/" element={<Home />} />
-		<Route path="/about/this/site" element={<About />} />
+		{/* <Route path="/about/this/site" element={<About />} /> */}
```
[the remark this answers is not in the document]
Comment (justsealan) [reply client/src/App.js:10]: comments removed

File: client/src/components/Jumbotron/HeroLayout.js (hunk 4 above)
Comment (justsealan) [reply client/src/components/Jumbotron/HeroLayout.js:65]: comments removed

File: client/src/components/Jumbotron/Registration.js (hunk 6 above)
Comment (justsealan) [reply client/src/components/Jumbotron/Registration.js:67]: comments removed

File: client/src/index.css (hunk 7 above)
Comment (justsealan) [reply client/src/index.css:1]: you can keep it

File: client/src/components/Jumbotron/Registration.js (hunk 5 above)
Comment (justsealan) [reply client/src/components/Jumbotron/Registration.js:19]: resolved in the next pr

**Review by Haseeb3927 — APPROVED** [review]
Approved

**Review by Haseeb3927 — APPROVED** [review]
Looks good. Let's see if it doesn't create any conflicts.

**Pull request merged**